### ISSUE TITLE
Hashtags and URLs not readable on Radiance and Ambiance theme on Lucid

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#195]
```
Hi,

if I use the Radiance or Ambiance theme on Ubuntu lucid the contrast
between hashtags and urls (grey) and the background (white) in pine isn't
really there anymore so it would be good if either the default colors get
changed or one would be able to set the colors onselves.

Bapf
```

Original issue reported on code.google.com by `bernd.st...@googlemail.com` on 7 Apr 2010 at 1:25

**Comment by GoogleCodeExporter** [comment]
```
Yep, this is something i would appreciate too ;)
```

Original comment by `mich.ba...@gmail.com` on 7 Apr 2010 at 6:39

**Comment by GoogleCodeExporter** [comment]
```
For fix this issue, must be use the property "link-color" and 
"visited-link-color" of
main GtkWidget which is better than the selected color on that of the main 
widget.

http://library.gnome.org/devel/gtk/stable/GtkWidget.html#GtkWidget--s-link-color
```

Original comment by `dna...@gmail.com` on 4 May 2010 at 3:29

**Comment by GoogleCodeExporter** [comment]
```
Ok. How can I get this property?
```

Original comment by `tro...@gmail.com` on 5 May 2010 at 11:18

**Comment by GoogleCodeExporter** [comment]
```
I don't know Vala language. This function in C returns the style of widget (you 
can
use the main window widget):

http://library.gnome.org/devel/gtk/stable/GtkWidget.html#gtk-widget-get-style

Then with the GtkStyle objects you can invoke this: 
gtk_style_get_style_property()

http://library.gnome.org/devel/gtk/stable/GtkStyle.html#gtk-style-get-style-prop
erty
```

Original comment by `dna...@gmail.com` on 5 May 2010 at 2:53

**Comment by GoogleCodeExporter** [comment]
```
Closed by a67d974f20 revision. Thanks.
```

Original comment by `tro...@gmail.com` on 6 May 2010 at 10:30
- Changed state: **Fixed**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**
